### DISCUSSION TITLE
Does Wails support in spellchecking in WebView

### DISCUSSION THREAD

**Discussion by idroz** [wailsapp/wails#2484] [Q&A] [answered]
I just noticed that a textarea in wails webview doesn't seem to support spell checking. Just wondering if there is a trigger to enable that or if the recommended way is to program it in?

**Comment by leaanthony** [comment, marked as answer]
It's not supported currently. Perhaps that could be something that can be added to v3.

**Reply by idroz** [reply]
Thanks - I think it would be a handy feature. For example, electron introduced it a few years ago ([electron spellchecker](https://github.com/electron/electron/pull/20692)). It uses the OS spellchecker on macOS and hunspell on all other platforms and is activated through the [BrowserWindow API](https://www.electronjs.org/docs/api/browser-window#new-browserwindowoptions). Perhaps a parallel here would be `wails.Run` API?

**Reply by leaanthony** [reply]
That appears to be using chrome APIs. That's not an option for us.
